tasktiger/task.py

```diff
@@ -142,6 +142,12 @@ def id(self):
     def data(self):
         return self._data
 
+    @property
+    def time_last_queued(self):
+        return datetime.datetime.utcfromtimestamp(
```

Comment:
Let's return `None` if `self._data` doesn't contain it (which it could if it's not a queued task).

Reply:
Thx, done